main.go

```diff
@@ -104,6 +104,7 @@ func main() {
 				GivenName:   name,
 				Idle:        config.Defaults.Idle,
 				CommandType: config.Defaults.CommandType,
+				PowerCondition: config.Defaults.PowerCondition,
```

Comment:
shouldn't `defaultConf` have the default value set in the struct in line 42? 
Otherwise, if `-p` is not given, the value will be `nil`.
Or am I missing something?